### ISSUE TITLE
Cannot get rid of device_tracker entities

### ISSUE THREAD

**Issue by ildar170975** [twrecked/hass-virtual#75]
Some time ago I created a few `device_tracker` entities by the Virtual integration.
Then I upgraded to the latest version 0.8.0.1.
Now I need to delete these entities.

What I did is:
1. Deleted the Virtual integration.
2. Uninstalled the Virtual Integration in HACS.
3. Removed these entities from "known_devices.yaml".
4. Removed these entities from yaml.
5. Restarted HA several times.

Nothing helps.
After next HA restart these entities again restored in "known_devices.yaml", again present in Settings -> Entities.

Searched in HA folder. These enities are present in "core.device_registry" as orphaned:
```
      {
        "config_entries": [],
        "connections": [],
        "identifiers": [
          [
            "virtual",
            "+iphone_5s_accum"
          ]
        ],
        "id": "035ee1b4246fae5ec50e81d75fb2a746",
        "orphaned_timestamp": 1699730700.0463932
      },
```
where `iphone_5s_accum` is a part of `device_tracker.iphone_5s_accum`.

**Comment by ildar170975** [comment]
Now I am not sure if it was 0.8.0.1 - it could be probably 0.9.x ....

**Comment by twrecked** [comment]
Sorry, it was the 0.9 upgrade. The code moves from the old device tracker mechanism to the current one. I don't want to automatically make changes to users' configuration files so there is some pain when upgrading this particular component. All other devices should work automatically.

I'm assuming these docs didn't work for you: https://github.com/twrecked/hass-virtual#what-you-need-to-be-wary-of

**Comment by ildar170975** [comment]
> 'm assuming these docs didn't work for you

Unfortunately - this did not help.
These entities stuck in setup.

**Comment by ildar170975** [comment]
Just updated to the latest 0.9.0.beta.11 from 0.8.x & started testing.

I had some amount of `device_tracker` entities in a separate file with a name like `trackers.yaml`:
```
device_tracker:
  - platform: virtual
    devices:

      - name: iphone_5s_accum
        peristent: True

      - name: redmi_7a_accum
        peristent: True

    ...
```
I did all steps described here: https://github.com/twrecked/hass-virtual?tab=readme-ov-file#what-you-need-to-be-wary-of, in particular:
1. Installed 0.9.
2. Renamed that `trackers.yaml` file to `trackers.dis`.
3. Commented all entries in `known_devices.yaml`.
4. Restarted HA.
5. Deleted the integration.
6. Added it again (it said it imported data from `virtual.yaml`).

To delete trackers I did these steps:
1. In `virtual.yaml` commented all trackers.
2. Deleted the integration.
3. Commented all entries in `known_devices.yaml`.
4. Restarted HA.
5. These trackers were NOT removed from the system.

**Comment by ildar170975** [comment]
So I assume that this was not fixed since the 1st 0.9 release.
Will have to use these stuck device_trackers. Not sure if they can be used.
